### PR TITLE
PYIC-3623: Commonise Experian KBV states

### PR DESCRIPTION
**Pull request by TalatAydinGDS**
### What changed

- added the ‘next’ event inside CRI_KBV_J2 as well, just like it is in CRI_KBV_J3
- renamed and merged PRE_KBV_TRANSITION_PAGE_J2 and PRE_KBV_TRANSITION_PAGE_J3 in the PRE_EXPERIAN_KBV_TRANSITION_PAGE

### Why did it change
   As far as we can see, everything is the same between PRE_KBV_TRANSITION_PAGE_J3, CRI_KBV_J3 AND PRE_KBV_TRANSITION_PAGE_J2, CRI_KBV_J2, except for the ‘next’ event.

### Issue tracking

- [PYIC-3623](https://govukverify.atlassian.net/browse/PYIC-3623)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

[PYIC-3623]: https://govukverify.atlassian.net/browse/PYIC-3623?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml [hunk 1]
```diff
@@ -490,10 +492,43 @@ CRI_HMRC_KBV_J6:
     end:
       targetState: IPV_SUCCESS_PAGE
     fail-with-no-ci:
-      targetState: PRE_KBV_TRANSITION_PAGE_J3
+      targetState: PRE_EXPERIAN_KBV_TRANSITION_PAGE
     next:
       targetState: PYI_NO_MATCH
 
+# Common pages
```
Comment (MikeCollingwood) [inline lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml:499]: Could these be put in the shared states at the top of this file?

File: lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml [hunk 2]
```diff
@@ -350,6 +350,8 @@ CRI_KBV_J2:
       checkIfDisabled:
         f2f:
           targetState: PYI_CRI_ESCAPE_NO_F2F
+    next:
```
Comment (MikeCollingwood) [inline lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml:353]: Not sure if this was needed as you're about to make the state redundant

**Review by MikeCollingwood — APPROVED** [review]
A couple of comments, but if you've tested this with the automated tests it LGTM! Though please wait until I can revert the breaking changes

File: lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml (hunk 1 above)
Comment (TalatAydinGDS) [reply lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml:499]: Yeah, good idea for reading easy. I will raise this change in the second PR.

File: lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml (hunk 2 above)
Comment (TalatAydinGDS) [reply lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml:353]: In the second PR, we will remove this, but I added it to ensure that the two pages are the same.

**Pull request merged**